### ISSUE TITLE
what is mean of '--steps'?

### ISSUE THREAD

**Issue by tianfengyijiu** [xuannianz/EfficientDet#23]
what mean of '--steps'?
why have two params 'steps' and 'epoch'?

**Comment by florisdesmedt** [comment]
The epochs used to be the number of times the training process iterates over the full dataset (all images in your training set). With the use of `--steps`, you limit the number of images processed in a single epoch to the number you pass (e.g. 1000).

A lot of callbacks, such as decreasing the learning rate when no improvement is obtained, or early-stopping, will handle on the start/end of epochs. Also, after each epoch, the model that is trained so far will be evaluated on your validation set.
